content/dstu2.md

```diff
@@ -21,7 +21,7 @@ All API access is over HTTPS. See [Service Root URL](#service-root-url) for more
 All data is sent and received as JSON.
 
 <pre class="terminal">
-$ curl -i -H "Accept: application/json+fhir" https://fhir-open.sandboxcernerpowerchart.com/dstu2/d075cf8b-3261-481d-97e5-ba6c48d3b41f/metadata
+$ curl -i -H "Accept: application/json+fhir" https://fhir-open.sandboxcerner.com/dstu2/d075cf8b-3261-481d-97e5-ba6c48d3b41f/metadata
```

Comment:
When do we want to update the tenant keys to FHIRPLAY? Should be easy but could potentially end up with some invalid example with a blanket find and replace. Thinking mostly of Patient parameter search and maybe the may2015 endpoint?